src/content/advanced/upgrades/aws-19-release/index.md

```diff
@@ -0,0 +1,124 @@
+weight: 10
+menu:
+  main:
+    parent: advanced-upgrades
+aliases:
+  - /guides/aws-release-v19/
+user_questions:
+  - Where can I read about v19 changes?
+  - What does v19 change?
+  - What changes does Cilium bring?
+  - What do i have to prepare for v19?
+  - Whad do i have to do before IRSA migration?
+  - How does the Cilium migration work?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-04-13
+---
+
+{{< platform_support_table aws="alpha=v19.0.0" >}}
+
+## Introduction
+
+We have been preparing for a long time to introduce release v19 on AWS. Besides upgrade of components and Kubernetes version to 1.24, this release will involve two major changes for customers i.e. migration from AWS CNI to Cilium and replacement of kiam with IAM Roles for Service Accounts(IRSA).
+Next sections are describing important changes we will introduce with the new release, the key benefits, what customers can do to prepare and how to avoid downtime during this crucial upgrade. 
+
+## Cilium
+
+Say goodbye to slow network initialization times and hello to lightning-fast performance with [Cilium](https://github.com/cilium/cilium), our new Kubernetes CNI solution!
+
+### Key Highlights
+
+- Service Mesh integration: `Cilium` is designed to work seamlessly with popular service meshes like `Istio`, `Linkerd`, and `Envoy`. This allows for more advanced networking and security features, such as mTLS encryption and observability.
+- `Cilium` uses a virtual network which provides more flexibility, faster network initialization, and more advanced networking features compared to `AWS CNI` affecting the IP addresses.
+- Advanced networking features: `Cilium` supports advanced networking features such as load balancing, network segmentation, and eBPF-based packet filtering. These features allow for more granular control over network traffic and improve security. 
+- Scalability: `Cilium's` eBPF-based data plane is highly scalable and performs well even at scale. It is also highly efficient, reducing overhead and maximizing performance.
+
+### What changes with Cilium?
+
+With `Cilium`, you'll no longer be using the `AWS CNI` Pod subnets, so be sure to add custom routes with the `Node subnet(s) CIDR(s)` instead. 
+
+Additionally, while `Cilium's Network Policy` provides powerful security features, support for setting `ipBlock` with `Pod IPs` is not included, so be sure to inspect your workloads and configured `Network Policies` carefully. The Account Engineers will reach out to you and we will provide a solution to provide the `NetworkCiliumPolicies` before the upgrade for no downtime.
+
+It's important to note that due to changes to `Cluster CR's` during the upgrade process, `GitOps` automation will have to be suspended and any applied changes backported to the repos before resuming. Keep this in mind as you prepare for the upgrade.
```

Comment:
```suggestion
It's important to note that due to changes to `Cluster CR's` during the upgrade process, `GitOps` automation will have to be suspended and any applied changes backported to the repos before resuming. Keep this in mind as you prepare for the upgrade. This needs to be evaluated on a case-by-case basis, since different GitOps implementations might only keep _some_ parts of `Cluster` CRs in Git. Feel free to reach out to your Account Engineer to understand more about these changes.
```